metro/proto/v1/spec.proto

```diff
@@ -778,3 +778,12 @@ message AdminTopic {
 
     int32 num_partitions = 3;
 }
+
+// Request made on the push endpoint
+message PushEndpointRequest {
```

Comment:
is this not defined in the google pubsub proto ?

Reply:
nope.